### PR TITLE
DA-794 fix pm status bug

### PR DESCRIPTION
**Pull request by Seeker1911**
add method to check for existing non-cancelled physical measurements  and only set PM.status to cancelled if it is the only measurement.

### CODE REVIEW

File: rest-api/dao/physical_measurements_dao.py [hunk 1]
```diff
@@ -298,8 +298,9 @@ def _update_participant_summary(self, session, obj):
     participant_summary.lastModified = clock.CLOCK.now()
 
     if obj.status and obj.status == PhysicalMeasurementsStatus.CANCELLED:
-      # update corresponding participant summary physicalMeasurementsStatus
-      participant_summary.physicalMeasurementsStatus = obj.status
+      if not self.existing_pm(session, participant):
```
Comment (calbach) [inline rest-api/dao/physical_measurements_dao.py]: Shouldn't this condition be inverted? I don't follow

Comment (Seeker1911) [reply rest-api/dao/physical_measurements_dao.py]: no. its "if you do not have an existing pm that is valid, set this to cancelled"

Comment (calbach) [reply rest-api/dao/physical_measurements_dao.py]: Got it now - maybe reword as `self.has_uncancelled_pm()`

File: rest-api/dao/physical_measurements_dao.py [hunk 2]
```diff
@@ -298,8 +298,9 @@ def _update_participant_summary(self, session, obj):
...
+        # update corresponding participant summary physicalMeasurementsStatus
+        participant_summary.physicalMeasurementsStatus = obj.status
```
Comment (calbach) [inline rest-api/dao/physical_measurements_dao.py]: opt: more characters, but might be slightly easier to understand if you replaced `obj.status` here with `PhysicalMeasurementsStatus.CANCELLED`

File: rest-api/dao/physical_measurements_dao.py (hunk 1 above)
Comment (Seeker1911) [reply rest-api/dao/physical_measurements_dao.py]: done

File: rest-api/dao/physical_measurements_dao.py (hunk 2 above)
Comment (Seeker1911) [reply rest-api/dao/physical_measurements_dao.py]: done
